src/Select/SelectInput.js

```diff
@@ -145,6 +149,7 @@ class SelectInput extends React.Component {
 
   render() {
     const {
+      autoFocus,
```

Comment:
This is breaking the native auto focus.

Reply:
Should i move it from props to somewhere else?

Reply:
You can remove this line or apply the `autoFocus` property to the select component.